codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/specs/ProtocolSpec.java

```diff
@@ -56,4 +61,21 @@ default CodeBlock asyncExecutionHandler(IntermediateModel intermediateModel, Ope
     default List<MethodSpec> additionalMethods() {
         return new ArrayList<>();
     }
+
+    default List<CodeBlock> errorUnmarshallers(IntermediateModel model, PoetExtensions poetExtensions) {
```

Comment:
I'd rename this method since it isn't error unmarshallers but rather a list of register statements.